conf/messages.en

```diff
@@ -672,10 +672,10 @@ individualRecipientNino.error.required = Select yes if you know the recipient’
 
 loanRecipient.companyRecipientCrn.title = Do you know the recipient’s Company Registration Number (CRN)?
 loanRecipient.companyRecipientCrn.heading = Do you know {0}’s Company Registration Number (CRN)?
-loanRecipient.companyRecipientCrn.hint = This is 7 or 8 characters, like 0123456 or AB123456. It may also be called ‘company number’ or ‘Companies House number’
+loanRecipient.companyRecipientCrn.hint = This is 8 characters, like AB123456. It may also be called ‘company number’ or ‘Companies House number’.
```

Comment:
Do we need the dot (.) at the end of this (and other) messages? Just to double check if this is not duplicated on the UI and shown as (..)